docs/operations/api-reference.md

```diff
@@ -293,10 +293,10 @@ Returns full segment metadata for a specific segment in the cluster.
 
 Return the tiers that a datasource exists in.
 
-#### Note for coordinator's POST and DELETE API's
-The segments would be enabled when these API's are called, but then can be disabled again by the coordinator if any dropRule matches. Segments enabled by these API's might not be loaded by historical processes if no loadRule matches.  If an indexing or kill task runs at the same time as these API's are invoked, the behavior is undefined. Some segments might be killed and others might be enabled. It's also possible that all segments might be disabled but at the same time, the indexing task is able to read data from those segments and succeed.
+#### Note for coordinator's POST and DELETE APIs
+The segments would be enabled when these APIs are called, but then can be disabled again by the coordinator if any dropRule matches. Segments enabled by these APIs might not be loaded by historical processes if no loadRule matches.  If an indexing or kill task runs at the same time as these APIs are invoked, the behavior is undefined. Some segments might be killed and others might be enabled. It's also possible that all segments might be disabled but at the same time, the indexing task is able to read data from those segments and succeed.
 
-> Caution : Avoid using indexing or kill tasks and these API's at the same time for the same datasource and time chunk. (It's fine if the time chunks or datasource don't overlap)
+> Caution : Avoid using indexing or kill tasks and these APIs at the same time for the same datasource and time chunk. (It's OK to use them if the time chunks or datasource don't overlap)
```

Comment:
We can remove the `Caution` label since this is already in a note block
```suggestion
> Avoid using indexing or kill tasks and these APIs at the same time for the same datasource and time chunk.
```